frontend/src/colors.js

```diff
@@ -35,6 +38,9 @@ export const eventTypeColors = {
   [CARD_STARTED]: colors.green[600],
   [PR_REVIEWED]: colors.purple[600],
   [COMPETENCE_REVIEW_DONE]: colors.blue[600],
+  [PROJECT_CARDS_COMPLETED]: colors.yellow[600],
+  [TOPIC_CARDS_COMPLETED]: colors.red[600],
```

Comment:
Please remove these changes.  None of the things you added here are event types.